cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -181,14 +182,30 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
     }
   }
 
+  private void sanityCheckSubmittedRequest(HttpServletRequest request, RequestInfo requestInfo) {
+    if (requestInfo.accessToAlreadySubmittedRequest()
+        && _userTaskManager.getUserTaskByUserTaskId(_userTaskManager.getUserTaskId(request), request) == null) {
+     throw new UserRequestException(
+         String.format("Attempt to start a new user task with an already submitted review. If you are trying to retrieve"
```

Comment:
IIUC, this Exception message seems to be confusing under certain scenario. 
Currently the default retention time for `Purgatory` is 14 days and the retention time for `UserTaskManager` is 1 day, so if a user want to retrieve (with UUID and reviewID)  a task which was submitted 2 days,  the user will get the this Exception and get confused(since he already provide the UUID).

Reply:
Please see [UserTaskManager#removeFromPurgatory](https://github.com/linkedin/cruise-control/pull/582/files#diff-e8b485e0404847263c842e3b93fb823aR301), which removes request from purgatory if the task expires.

Let me know if I am missing something.